tests/acceptance/tests/Settings/WishlistGuestFunctionality.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { test } from '@fixtures/AcceptanceTest';
+
+test('Customer is able to add and remove products to the wishlist',{ tag: '@Wishlist' }, async ({ 
+    TestDataService,
+    ShopCustomer,
+    StorefrontHome,
+    AddProductToWishlist,
+    StorefrontWishlist,
+    AddProductToCartFromWishlist,
+    Login
+}) => {
+    await TestDataService.setSystemConfig({ 'core.cart.wishlistEnabled': true });
+    const product1 = await TestDataService.createBasicProduct();
+    const product2 = await TestDataService.createBasicProduct();
+    const product1Locators = await StorefrontHome.getListingItemByProductId(product1.id);
+    const product2Locators = await StorefrontHome.getListingItemByProductId(product2.id);
+
+    await test.step('Add to products to the wishlist and check the basket count', async () => {
```

Comment:
You don't add products in this step.
```suggestion
    await test.step('Accept all cookies and reload page', async () => {
```

Reply:
Should be fixed 👍